unittests/scans/govulncheck/many_vulns_new_version.json

```diff
@@ -1,267 +1,194 @@
 [
```

Comment:
Hi @a-ruff ,

I would prefer if you would keep the already existing unittests report files and add an additional one if you advance govulncheck.

Reply:
+1

Reply:
Sure, I'll proceed with the change.

Reply:
Done ✅